package-sets/1.0.0.json

```
@@ -5,66 +5,66 @@
   "packages": {
     "arrays": {
       "git": "https://github.com/purescm/purescript-core.git",
-      "ref": "98f12cc76d4941f979f2530da94f5b46ce9551cd",
+      "ref": "1982feebeb96103c9e8caaf6a646498445e61cb5",
```

If we used a yaml reference we could have this hash in a single place
```suggestion
      "ref": &core-ref "1982feebeb96103c9e8caaf6a646498445e61cb5",
```

See e.g. how's it done [here](https://github.com/purescript/spago/issues/1295#issuecomment-2450846910)

---

But this is not yaml. :)